### PR TITLE
doctl-auth: add page

### PR DESCRIPTION
**Pull request by 258204**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).
For #7475

### CODE REVIEW

File: pages/common/doctl-auth.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# doctl auth
+
+> Authenticate doctl with one or more API tokens.
+> More information: <https://docs.digitalocean.com/reference/doctl/reference/auth/>.
+
+- Show available commands:
+
+`doctl auth --help`
```
Comment (sbrl) [inline pages/common/doctl-auth.md:8]: Examples in tldr pages should be ordered in the order people might use them. Let's put this example at the bottom of the page.

File: pages/common/doctl-auth.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- List authentication contexts (API tokens):
+
+`doctl auth list`
+
+- Switch contexts (API tokens):
+
+`doctl auth switch --context {{token_label}}`
+
+- Remove a context (API token):
```
Comment (sbrl) [inline pages/common/doctl-auth.md:22]: Not sure I understand the `(API token)` in brackets here. Can we reword this to improve clarity? Same goes with the example above this one.

**Review by sbrl — COMMENTED** [review]
Thanks for the new page! I've left some comments below for you to review.

File: pages/common/doctl-auth.md (hunk 2 above)
Comment (258204) [reply pages/common/doctl-auth.md:22]: That's a good point. In Digital Ocean, when you generate an API token, enter it into doctl, and give it a label, it creates a context.

When switching contexts you are switching between stored API tokens. Does the following read, better, or is there a better way to word this?
`- Remove a context (a locally stored API token):`

Comment (258204) [reply pages/common/doctl-auth.md:18]:
```suggestion
- Remove a stored authentication context (API token):
```
Does this illustrate what I wrote above?

Comment (sbrl) [reply pages/common/doctl-auth.md:18]: Ah, I see! Thanks for clarifying. Yeah, @258204 - that looks good!

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @258204!
